specification/electronic-prescription-service-api.yaml

```diff
@@ -340,10 +340,10 @@ paths:
             examples:
               return:
                 value:
-                  $ref: components/examples/secondary-care/community/acute/nominated-pharmacy/clinical-practitioner/1-Task-Request-Return-200_OK.json
+                  $ref: components/examples/secondary-care/homecare/acute/nominated-pharmacy/clinical-practitioner/2-Task-Request-Return-200_OK.json
```

Comment:
Moved to homecare for testing dispense scenarios, instead of moving do we want both community example and homecare examples for these? If so, which one or should we reference both in spec?

Reply:
discussed with Dan, homecare examples sufficient for our current needs